src/commandDetails/miscellaneous/info.ts

```diff
@@ -20,7 +20,7 @@ const getInfoEmbed = async (): Promise<MessageEmbed> => {
     .setURL(githubRepositoryInfo.html_url)
     .setThumbnail(githubRepositoryInfo.owner.avatar_url)
     .setDescription(
-      'Links to issue templates: https://github.com/uwcsc/codeybot/tree/main/.github/ISSUE_TEMPLATE',
+      'Links to issue templates: https://github.com/uwcsc/codeybot/issues/new?assignees=&labels=request&template=feature_request.md&title=%5BFEATURE%5D',
```

Comment:
Would it be possible to also change `Links to issue templates` to `Make a feature request`?